meadow-client/src/MainFrame.purs

```diff
@@ -578,6 +563,7 @@ mainTabBar activeView = navTabs_ (mkTab <$> tabs)
   tabs =
     [ Editor /\ "Haskell Editor"
     , Simulation /\ "Simulation"
+    , BlocklyV /\ "Blockly"
```

Comment:
Why the `V`? Tempting to go for `HaskellEditor | Simulation | BlocklyEditor`?

Reply:
yeah that was a temporary name, I meant to revisit it. Your name is nice, have used.